geocoder/extension/sql/40_namedplaces_test.sql

```diff
@@ -0,0 +1,20 @@
+-- Check that the different geocoding functions are callable, should return success = false
+SELECT (geocode_namedplace(Array['Madrid', 'New York City', 'sunapee'])).*;
+SELECT (geocode_namedplace(Array['Elche', 'Granada', 'Madrid'], 'Spain')).*;
+SELECT (geocode_namedplace(Array['sunapee', 'sunapeeee', 'New York City', 'Madrid'], Array['', 'US', 'United States', NULL])).*;
+SELECT (geocode_namedplace(Array['Portland', 'Portland', 'New York City'], Array['Maine',    'Oregon',    NULL], 'USA')).*;
+SELECT (geocode_namedplace(Array['Portland'], 'Oregon', 'USA')).*;
+SELECT (geocode_namedplace(Array['Portland', 'Portland', 'New York City'], Array['Maine',    'Oregon',    NULL], Array['USA'])).*;
+
+-- Add a named place source
+COPY global_cities_alternates_limited (geoname_id, name, the_geom, created_at, updated_at, the_geom_webmercator, preferred, lowername, cartodb_id, admin1_geonameid, iso2, admin1) FROM stdin;
```

Comment:
Did these COPY work? It could be worthy to throw an INSERT INTO here, just to avoid the issues we've suffered already with the formats, but we can do this after merge.